### ISSUE TITLE
Full Request/Response not logged because of missing TeeFilter

### ISSUE THREAD

**Issue by donovanmuller** [akkinoc/logback-access-spring-boot-starter#34]
As far as I can see (may be mistaken) there is no configuration for the Logback [`TeeFilter`](https://logback.qos.ch/access.html#teeFilter).

So if you want the full request/response in your logging, for example:

`logback-access.xml`
```xml
<configuration>
    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
        <encoder>
            <pattern>%fullRequest%n%n%fullResponse</pattern>
        </encoder>
    </appender>
    <appender-ref ref="CONSOLE" />
</configuration>
```

you will always see a blank [`responseContent`](https://github.com/qos-ch/logback/blob/master/logback-access/src/main/java/ch/qos/logback/access/spi/AccessEvent.java#L524).

This is because the `AccessConstants.LB_OUTPUT_BUFFER` attribute is never set by the [`TeeFilter`](https://github.com/qos-ch/logback/blob/master/logback-access/src/main/java/ch/qos/logback/access/servlet/TeeFilter.java#L59).

The workaround is to manually configure a `TeeFilter` something like:

```java
@Bean
public FilterRegistrationBean requestLoggingFilter() {
    FilterRegistrationBean filterRegistrationBean = new FilterRegistrationBean(new TeeFilter());
    // filterRegistrationBean.setInitParameters(...);
    return filterRegistrationBean;
}
```

and the request/response is logged.

_Is omitting the `TeeFilter` auto configuration intentional or something that should be added?_

**Comment by akihyro** [comment]
Thanks for report!

Exactly, I currently do not provide TeeFilter.
I think there is no problem with your workaround.

I 'd like to support in a future versions.
I'm thinking to provide TeeFilter by Spring property (application.yml, etc).
for example:

```yml
logback.access:
  teeFilter:
    enabled: true
    includes: ...
    excludes: ...
```

Thanks.

**Comment by akkinoc** [comment]
Released in v2.10.0!
